client/src/components/LinkButtons.tsx

```diff
@@ -17,58 +17,69 @@ const ButtonRow = styled('div')({
   display: 'flex',
   justifyContent: 'center',
   flexWrap: 'wrap',
-  gap: '50px',
 });
 
 interface IconButtonData {
   link: string;
   icon: React.ReactElement;
-  name: string;
+  name: string | undefined;
```

Comment:
Why do we need `undefined` here?

Reply:
This is needed since we decided to leave the NO_ICON (in LinkIconsLib.tsx) with the undefined name.

Reply:
We talked about using `topmenu` as names / prefix for the icons. Please use `code-github` and `help` as icon names.